tests/shinytest/UI_tests-expected/022.json

```diff
@@ -1,9 +1,9 @@
 {
   "output": {
     "geog_agg2": [
-      "You have at least one hyphen in your variable names, you need to remove all hyphens from variable names to use this part of the app.",
+      "An error has occurred. Check your logs or contact the app author for clarification.",
```

Comment:
Correct error message shows when I run the shinytest file through the app - is it just showing this because we've overwritten this default message?
![image](https://user-images.githubusercontent.com/75032474/128315528-9edf9a70-b111-4fc9-abfd-1b5b917899bc.png)

Reply:
Yeah, shiny test isn't really doing much for these new error messages now which is frustrating. I had a look through all the outputs they take but couldn't find anything useful so just left it as this for now, I've added something to trello to look into changing our testing around the QA tabs, maybe looking at rselenium or using specific screenshots.